rust-symcrypt/src/lib.rs

```diff
@@ -3,25 +3,21 @@
 //! This crate provides friendly and idiomatic Rust wrappers over [SymCrypt](https://github.com/microsoft/SymCrypt), an open-source cryptographic library.
 //!
 //! This crate has a dependency on `symcrypt-sys`, which utilizes `bindgen` to create Rust/C FFI bindings.
-//!
-//! **Note:** As of version `0.1.3`, only `Windows AMD64`, and [`Linux Mariner`](https://github.com/microsoft/CBL-Mariner) are fully supported, with partial support for other linux distros such as `Ubuntu`.
-//!
-//! For ease of use, we have included a `symcrypttestmodule.dll` and `symcrypttestmodule.lib` in the `C:\Users\<your-user>\.cargo\registry\src\github.com-****\symcrypt-0.1.3\bin\amd64` folder for users on Windows. This will only work on computers using an `AMD64 (x86_64)` architecture.
-//!
-//! We have also included the required `libsymcrypt.so` files needed for Linux users. These `.so` files have been built for the [Mariner](https://github.com/microsoft/CBL-Mariner) distro, but have been tested and confirmed working on `Ubuntu 22.04.3` via WSL. Support for other distros aside from Mariner is not guaranteed. 
-//! These files are included in the `~/.cargo/registry/src/github.com-****/symcrypt-0.1.3\bin\linux` folder for users on Linux. 
 //! 
-//! If you are using a different architecture, you will have to continue with the install and build steps outlined in the `BUILD.md` file.
-//!
+//! **Note:** As of version `0.2.0`, only `Windows AMD64`, and [`Linux Mariner`](https://github.com/microsoft/CBL-Mariner) are fully supported, with partial support for other linux distros such as `Ubuntu`.
```

Comment:
>//! **Note:** As of version `0.2.0 [](http://example.com/codeflow?start=0&length=34)

To make this easier to maintain, I'd recommend removing all of the duplicated README content and just pointing to the README/BUILD files.

Reply:
This is the documentation that shows up in the rust docs like this: https://docs.rs/symcrypt/0.1.3/symcrypt/

If you think that it would be better, we just point to a readme we can do that, but from what i've seen from other rust packages they have a similar layout in the `lib.rs` example rustls: https://docs.rs/rustls/0.23.4/rustls/

Reply:
This looks good to me. Same comment about "Linux Azure Linux" though

Reply:
Fixed